docs/azure_arc_jumpstart/azure_arc_data/aks/aks_mssql_mi_ad_auth_arm_template/_index.md

```diff
@@ -0,0 +1,224 @@
+---
+type: docs
+title: "SQL Managed Instance with AD Authentication ARM Template"
+linkTitle: "SQL Managed Instance with AD Authentication ARM Template"
+weight: 2
+description: >
+---
+
+## Deploy Azure SQL Managed Instance with AD authentication support in directly connected mode on AKS using an ARM Template
```

Comment:
Point out that this scenario is in customer-managed keytab mode

Reply:
Done